base/scripts/objectives.cs

```diff
@@ -1013,17 +1056,22 @@ function checkObjectives(%this)
           
           $FlagIsDropped[%playerTeam] = false;
           $FlagIsDropped[%enemyTeam] = false;
-          
+
           $freeze::FlagClient[%playerTeam] = 0;
           $freeze::FlagClient[%enemyTeam] = 0;
           
           $freeze::OOB[%playerTeam] = false;
           $freeze::OOB[%enemyTeam] = false;
+          
+          Observer::CheckFlagStatus(0);
+          Observer::CheckFlagStatus(1);
```

Comment:
technically we only need to check the flag was captured right? could we do that to save some cycles?

Reply:
correct.